### DISCUSSION TITLE
What are your favorite samples?

### DISCUSSION THREAD

**Discussion by ghost** [kaythomas0/noisedash#22] [General]
Where do you get your samples from? What are the ones that you use the most?

**Comment by kaythomas0** [comment]
Hey @BetaPictoris! I really like https://freesound.org/ It has hundreds of thousands of sounds and millions of registered users. All the sounds are free to download (you just need to make a free account). I've gotten some nice rain, fire, thunder, and ambient-cafe noises from there that work great with Noisedash :)

**Reply by ghost** [reply]
Hi @kaythomas0, amazing work on the project! I've been getting my loops from AmbientMixer, inspect -> network -> XHR, and I haven't really seen this site, I'll defiantly use it in the future, so thank you!
